alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultNotificationManager.java

```diff
@@ -89,6 +89,7 @@ public DefaultNotificationManager(NotificationContentRepository notificationCont
                                                        .stream()
                                                        .map(this::toModel)
                                                        .collect(Collectors.toList());
+        notificationContentRepository.flush();
```

Comment:
Can we also add a TODO to move the event sending out of the database api?

Reply:
I agree with Gavin, EventManager should be pulled out of the the NotificationManager. That would allow the transaction to save the notifications to complete before sending the events and would remove the need for the flush. But that can be done as a future improvement.

Reply:
I added the TODO.  This morning I tried refactoring the NotificationManager to refactor the transaction boundaries.  But that wasn't working so I opted for the simple approach for now.  This should be refactored in the future.